pkg/config/remote/service/service.go

```diff
@@ -211,6 +182,7 @@ type uptaneClient interface {
 	StoredOrgUUID() (string, error)
 	Targets() (data.TargetFiles, error)
 	TargetFile(path string) ([]byte, error)
+	TargetFiles(files []string) (map[string][]byte, error)
```

Comment:
is it worth removing the "single" call just above this?

Reply:
> Removing the "single" call

We could, it's not used anymore!

> file string...

I don't think the syntactic sugar here really does much. With how we use it we're always building a []string first. So there really is no "single or multiple" use case here to support. If this was a public library maybe for flexibility at the call site, but since that's not the case.

Reply:
Then let's remove it, less code to deal with